gradle/libs.versions.toml

```
@@ -73,6 +73,10 @@ slf4j-simple = "2.0.16"
 github-kotlin-logging = "7.0.3"
 okhttp = "4.12.0"
 
+os-detector = "1.7.3"
+download = "5.6.0"
+helmDocs = "1.14.2"
```

```suggestion
helm-docs = "1.14.2"
```

Our convention is to use dashes instead of camelCase right?